sorting/inserion.js

```diff
@@ -0,0 +1,14 @@
+function sorting(arr) {
```

Comment:
All good. Just change the name of function to `sort` or better `insertionSort(arr)`

Once done, merge